docs/user/adding-new-metrics.md

```diff
@@ -6,8 +6,7 @@ When adding a new metric, the workflow is:
 * Add a new entry to [`metrics.yaml`](#The-metricsyaml-file).
 * Add code to your project to record into the metric by calling the Glean SDK.
 
-> **Important**: Any new data collection requires documentation and data-review.
-This is also required for any new metric automatically collected by the Glean SDK.
+> **Important**: Any new data collection requires documentation and data-review. This is also required for any new metric automatically collected by the Glean SDK.
```

Comment:
Do we link to our data collection policy anywhere?  Would here be a good spot to do that?

Reply:
We do in a bunch of other places, but I agree, let's link to it from here, too.